format-specs/geoparquet.md

```diff
@@ -79,6 +80,15 @@ This is the binary format that the geometry is encoded in. The string 'WKB' to r
 [Well Known Binary](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary) is the only current option, but future versions
 of the spec may support alternative encodings. This should be the ["standard"](https://libgeos.org/specifications/wkb/#standard-wkb) WKB representation.
 
+#### edges
+
+This attribute indicates the coordinate system to be applied when interpreting the edges. Available values are:
+- planar: use flat cartesian coordinate system.
+- spherical: use spherical coordinate system and radius derived from the spheroid defined by the SRID.
+- spheroid: use an spheroid or ellipsoid defined by the SRID.
```

Comment:
+1 - just changed my suggestion to be spheroidal (since I think if we go sphere and spheroid then we also should say 'plane' instead of planar?)

Reply:
At some points we mentioned "let's leave out spheriod / ellipsoid (for 0.1) until we actually have a use case / someone asking for it" during the meeting, but I don't know if we made a clear decision on that.

Reply:
Agree with the new naming suggestion @paleolimbot @cholmes.

Regarding the inclusion of `spheroidal` in the 0.1 @jorisvandenbossche: we didn't make a final decision on that. I included it for completeness and because it's an informative attribute, but if we all agree on leaving it out of version 0.1, I will apply the changes in the branch.

Reply:
> As a note, BigQuery already supports spheroidal precision in the GIS functions, for example [ST_DISTANCE](https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#st_distance).

That is for an operation, and not for IO. When reading data, it seems they don't make a distinction between spherical or spheroidal (see https://cloud.google.com/bigquery/docs/geospatial-data#coordinate_systems_and_edges, which mentions a planar=True/False flag for [ST_GEOGFROMTEXT](https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#st_geogfromtext_signature2))

Reply:
I think we should leave it out for 0.1. It's easy to add it in when there's a use case. But it's hard to remove it, since we can't be sure that no one used it and relies on it after we put this out. I think we should open an issue for it, show how we're planning to represent it and ask for feedback. Then hopefully people will easily google it and can comment on their use case if they're looking for it.

Reply:
Agree with both. Regarding IO BigQuery distinguishes between planar (GeoJSON or WKT+planar flag) or spherical (WKT by default).

I think this opens the debate of using boolean vs string, so planar flag vs edges enum.